manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "dxt_version": "0.1",
+  "name": "website-generator",
+  "display_name": "Website Generator",
+  "version": "1.0.3",
+  "description": "Website Generator creates a professional, engaging, and user-friendly website for your business in seconds using AI.",
```

Comment:
Should I change it to this instead? --> 🔮 Create a website in seconds! 🔮 Generate, design, write code, and write copy for your website. Powered by B12. 📧hello@b12.io 📧